app/src/main/java/org/oppia/app/settings/profile/ProfileRenameViewModel.kt

```diff
@@ -8,5 +8,10 @@ import javax.inject.Inject
 /** The ViewModel for [ProfileRenameActivity]. */
 @ActivityScope
 class ProfileRenameViewModel @Inject constructor() : ObservableViewModel() {
+  val inputName = ObservableField("")
   val nameErrorMsg = ObservableField("")
+
+  fun sas(){
```

Comment:
Can you make this method name more contextual.

Reply:
Removed